frontend/src/components/organisms/Table/Table.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Header from '../../atoms/Header/Header';
+import TableRow from '../../molecules/TableRow/TableRow';
+import TableHeader from '../../molecules/TableHeader/TableHeader';
+import TableCell from '../../atoms/TableCell/TableCell';
+
+const Wrapper = styled.div`
```

Comment:
Remove default table style. Table should be without border and with given shadows.
![image](https://user-images.githubusercontent.com/33421875/117198499-63e6fe00-ade9-11eb-8947-cedff8a91eff.png)

Reply:
removed